olive/evaluator/olive_evaluator.py

```diff
@@ -313,6 +353,22 @@ def compute_latency(metric: Metric, latencies: Any) -> MetricResult:
             )
         return MetricResult.parse_obj(metric_res)
 
+    @staticmethod
+    def compute_throughput(metric: Metric, latencies: Any) -> MetricResult:
+        """Compute latency metrics."""
+        latency_metrics = OliveEvaluator.latency_helper(latencies)
+        metric_res = {}
+        batch_size = metric.user_config.batch_size
+        for sub_type in metric.sub_types:
+            latency_sub_type_name = LatencySubType.MAX if sub_type.name == ThroughputSubType.MIN else LatencySubType.MIN
```

Comment:
Do we only have 2 throughput metrics? What about other subtypes for throughput?

Reply:
Good catch. Throughput has similar sub types with latency, but with different `higher_is_better`. For example:
1. for max throughput, we should use batch_size / latency_min.
2. for min throughput, we should use batch_size / latency_max.
3. for percentile@90 of thoughput(higher -> lower), we should use batch_size / latency_percentile@90(lower->higher)

Reply:
Updated.

Reply:
Can you explain more about how to define "throughput" here? In which case for throughput "lower is better"?

Reply:
Throughput is always `higher is better`, that says `higher_is_better == True`. But for latency `higher_is_better == False`

> hroughput has similar sub types with latency, but with different higher_is_better

Reply:
If so, except `avg`, for other `Pn` throughput, I don't think it matches latency `Pn` metric. Should it be something like Latency `Pn` -> Throughput `P(100-n)`?

Reply:
Good question, but I think it is fine. 
latency P(75) means, 75% cases, the latencies number <= the value P(75). 
When it switches to throughput, it means 75% of cases, the throughput number >= the value P(75). 
The P(n) should always be the most bad case of n%. (but the `bad` for latency is higher, for throughput is lower)

Reply:
This makes sense to me. Thanks!

Reply:
Thanks!